public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,196 +224,208 @@ figure
-<div style="margin-left:30px">
-code-example(language="html").
-    &lt;hero-list>&lt;/hero-list>
-</div>  
-:marked
-  >Angular inserts an instance of the `HeroListComponent` view between those tags.
-  
-  * `templateUrl` - the address of this component's template which we showed [above](#template).
-  
-  * `directives` - an array of the Components or Directives that *this* template requires.
+  Here we see the `@Component` !{_decorator} which (no surprise) identifies the class
+  immediately below it as a component class.
+
+block ts-decorator
+  :marked
+    A decorator is a function. Decorators often have a configuration parameter.
+    The `@Component` decorator takes a required configuration object with the
+    information Angular needs to create and present the component and its view.
+
+    Here are a few of the possible `@Component` configuration options:
+
+:marked
+  - `selector`: CSS selector that tells Angular to create and insert an instance of this component
+  where it finds a `<hero-list>` tag in *parent* HTML.
+  For example, if an app's  HTML contains `<hero-list></hero-list>`, then
+  Angular inserts an instance of the `HeroListComponent` view between those tags.
+
+  - `templateUrl`: address of this component's template, which we showed [above](#templates).
+
+  - `directives`: !{_array} of the components or directives that *this* template requires.
   We saw in the last line of our template that we expect Angular to insert a `HeroDetailComponent`
-  in the space indicated by `<hero-detail>` tags. 
-  Angular will do so only if we mention the `HeroDetailComponent` in this `directives` array.
-  
-  * `providers` - an array of **dependency injection providers** for services that the component requires.
-  This is one way to tell Angular that our component's constructor requires a `HeroService` 
-  so it can get the list of heroes to display. We'll get to dependency injection in a moment.
+  in the space indicated by `<hero-detail>` tags.
+  Angular will do so only if we mention the `HeroDetailComponent` in this `directives` !{_array}.
+
+  - `providers`: !{_array} of **dependency injection providers** for services that the component requires.
+  This is one way to tell Angular that our component's constructor requires a `HeroService`
+  so it can get the list of heroes to display. We'll get to dependency injection later.
+
 figure
   img(src="/resources/images/devguide/architecture/template-metadata-component.png" alt="Metadata" align="left" style="height:200px; margin-left:-40px;margin-right:10px" )
+
 :marked
-  The `@Component` function takes the configuration object and turns it into metadata that it attaches
-  to the component class definition. Angular discovers this metadata at runtime and thus knows how to do "the right thing".
-  
-  The template, metadata, and component together describe the view.
+  At runtime, Angular discovers the metadata specified by the `@Component`
```

Comment:
Ok, but changed "discovers" to "reads" (otherwise the phrase doesn't really make sense).